projects/plugins/jetpack/_inc/lib/markdown/extra.php

```diff
@@ -1,70 +1,123 @@
-<?php
-#
-# Markdown Extra  -  A text-to-HTML conversion tool for web writers
-#
-# PHP Markdown & Extra
-# Copyright (c) 2004-2013 Michel Fortin
-# <http://michelf.ca/projects/php-markdown/>
-#
-# Original Markdown
-# Copyright (c) 2004-2006 John Gruber
-# <http://daringfireball.net/projects/markdown/>
-#
-# Tweaked to remove WordPress interface
-
-
-define( 'MARKDOWN_VERSION',  "1.0.2" ); # 29 Nov 2013
-define( 'MARKDOWNEXTRA_VERSION',  "1.2.8" ); # 29 Nov 2013
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
```

Comment:
Since this is a copy-pasted external library, maybe we should just add it to a `.phpcsignore` instead of reformatting it.

Or even better, maybe we should just pull in [michelf/php-markdown](https://packagist.org/packages/michelf/php-markdown) via Composer. But that's probably better left to a different PR.

Reply:
I was thinking about both of those options as well. Unfortunately, since our own coding standards are quite different from the ones used there, I'm not sure our changes will be useful there.

I'm happy to create a new `projects/plugins/jetpack/.phpcsignore` for this one; should I go ahead and cancel my changes here?

Reply:
> I'm happy to create a new `projects/plugins/jetpack/.phpcsignore` for this one; should I go ahead and cancel my changes here?

:+1: That's what I recommend.

> Unfortunately, since our own coding standards are quite different from the ones used there, I'm not sure our changes will be useful there.

I think you misunderstood something. If we've made actual changes (not just phpcs reformatting) locally and want to submit them upstream, we should follow upstream's coding standards when doing so. But I don't know if we have, so I didn't recommend pushing anything upstream.

What I did recommend (for a separate PR, not this one) was looking at pulling in the upstream code via composer instead of having a copy-paste of it in here.

Reply:
In 39eda4b2919f599856ed88bc31bc19c0a3ff4b58 I've reverted my changes and added a `.phpcsignore` file as suggested.